.github/copilot-instructions.md

```diff
@@ -0,0 +1,90 @@
+# Copilot Instructions for Sous Chefs Cookbooks
+
+## Repository Overview
+
+**Chef cookbook** for Percona MySQL Server, client tools, XtraBackup, Toolkit, and XtraDB Cluster. Representative of Sous Chefs cookbook ecosystem.
+
+**Key Facts:** Ruby-based, ~800KB, 46 Ruby files, Chef >= 16 required, supports CentOS 7+/Debian 10+/Ubuntu 18.04+
```

Comment:
Key facts: Ruby-based, Chef >- 16 required, supports various OS platforms (check metadata.rb, kitchen.yml and .github/workflows/ci.yml for which platforms to specifically test)